components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/modules/oauth/token-protected-service-invokers.js

```diff
@@ -80,9 +81,10 @@ var invokers = function () {
 
         if (devicemgtProps["isOAuthEnabled"]) {
             var accessToken = privateMethods.getAccessToken();
-            if (!accessToken) {
+            if (accessToken == null) {
                 userModule.logout(function () {
                     response.sendRedirect(devicemgtProps["appContext"] + "login");
+                    return responseCallback({"status": 401, "responseText" : "Session expired!"});
```

Comment:
Calling a responseCallback() is irrelevant here as there is no actual call made to a backend API at this moment and no actual backend response is retrieved